src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -61,7 +61,15 @@ try {
                       }
                   }
               },
-              error -> Log.e("AuthQuickstart", "SignIn failed: " + error)
+              error -> {
```

Comment:
I dont think this is a docs issues. Shouldn't we respond with `CONFIRM_SIGN_UP` step when Cognito returns `UserNotConfirmedException`? 

Can we verify what is the behavior on iOS and Amplify v1?

Reply:
You're correct in that we _should_ respond that way. But Amplify Android has had the incorrect behaviour for a long time - including in V1 - and we cannot change it now without potentially breaking customers' apps.

We will fix this properly in V3.